src/views/tutorial-view/components/image-config/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactElement, ReactNode } from 'react'
+import getValidatedImgChild from './get-validated-image-child'
+import MdxImg from '../mdx-img'
+// import classNames from 'classnames'
+import s from './image-config.module.css'
+
+/**
+ * Configure extra settings for images via MDX.
+ * Designed to wrap a single markdown image.
+ */
+export default function ImageConfig({
+  /** Hide borders (temporarily configurable while we remove baked-in borders from existing images) */
+  hideBorder = false,
+  /** A caption to display below the image. */
+  caption,
+  /** An MDX `img`, e.g. `![some alt text](path/to/img.jpg)` */
+  children,
+}: {
+  caption?: string
+  hideBorder?: boolean
+  children: ReactNode
+}): ReactElement {
```

Comment:
Could you move the comments on these props and the props interface definition to a new `types.ts` file local to this component?